ci/test_wheel.sh

```diff
@@ -10,7 +10,7 @@ package_name="libucx"
 
 WHEELHOUSE="${PWD}/dist/"
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
-RAPIDS_PY_WHEEL_NAME="ucx_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 cpp "${WHEELHOUSE}"
+WHEELHOUSE=$(RAPIDS_PY_WHEEL_NAME="ucx_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-github cpp)
```

Comment:
We're defining `WHEELHOUSE` twice? Something seems off here.

Reply:
ah! you're right, the first one should be removed, in favor of using the temporary directory created + returned by `rapids-download-wheels-from-github`.

Fixed in https://github.com/rapidsai/ucx-wheels/pull/23/commits/f8d766a5b782963629e4030aa2cb9615659134b9